x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/steps/data_stream_step/data_stream_step.tsx

```diff
@@ -99,9 +100,12 @@ export const DataStreamStep = React.memo<DataStreamStepProps>(
           setIntegrationValues({ dataStreamDescription: e.target.value }),
         inputTypes: (options: EuiComboBoxOptionOption[]) => {
           setIntegrationValues({ inputTypes: options.map((option) => option.value as InputType) });
+          setHasCelInput(
+            options.map((option) => option.value as InputType).includes('cel' as InputType)
```

Comment:
May be add a comment about using the value from https://github.com/elastic/kibana/pull/196356/files#diff-a89884afb36cc67c03c957120ac4ba2c5141210665f75564d87d8ebdfcc2ef39R34 just to not change / remove this accidentally.